### ISSUE TITLE
[Electra] Unable to spawn apps using dyld3, i.e. system apps

### ISSUE THREAD

**Issue by ChiChou** [frida/frida-core#176]
Good job for supporting Electra Jailbreak!

But spawn functionality looks still broken.

```
➜  gui git:(entitle) ✗ frida -U -f com.apple.DocumentsApp
     ____
    / _  |   Frida 10.6.55 - A world-class dynamic instrumentation toolkit
   | (_| |
    > _  |   Commands:
   /_/ |_|       help      -> Displays the help system
   . . . .       object?   -> Display information about 'object'
   . . . .       exit/quit -> Exit
   . . . .
   . . . .   More info at http://www.frida.re/docs/home/
Failed to spawn: unexpectedly timed out while waiting for process to suspend
```

**Comment by csftech** [comment]
@ChiChou
It looks like you are using an older version of Frida.
Upgrade Frida to 10.7 and try it again.

**Comment by ChiChou** [comment]
@csftech 

```
➜  gui git:(fix/null_is_gone) frida -U Files
     ____
    / _  |   Frida 10.6.55 - A world-class dynamic instrumentation toolkit
   | (_| |
    > _  |   Commands:
   /_/ |_|       help      -> Displays the help system
   . . . .       object?   -> Display information about 'object'
   . . . .       exit/quit -> Exit
   . . . .
   . . . .   More info at http://www.frida.re/docs/home/

[iPhone::Files]-> Frida.version
"10.7.0"
```

**Comment by oleavr** [comment]
@ChiChou Which version of Electra do you have? Tweaks on or off?

**Comment by ChiChou** [comment]
Electra 1.0.4 with tweaks on

**Comment by oleavr** [comment]
@ChiChou Perfect, that's the exact same configuration that I'm using. So your issue is actually just about Apple apps – App Store apps can be spawned just fine.

**Comment by oleavr** [comment]
Fix released in Frida 10.7.1.
